algorithmic_efficiency/workloads/fastmri/workload.py

```diff
@@ -9,13 +9,20 @@
 
 class BaseFastMRIWorkload(spec.Workload):
 
-  def has_reached_goal(self, eval_result: float) -> bool:
-    return eval_result['validation/ssim'] > self.target_value
+  def has_reached_validation_target(self, eval_result: float) -> bool:
+    return eval_result['validation/ssim'] > self.validation_target_value
 
   @property
-  def target_value(self) -> float:
+  def validation_target_value(self) -> float:
     return 0.7344
 
+  def has_reached_test_target(self, eval_result: float) -> bool:
+    return eval_result['test/ssim'] < self.test_target_value
```

Comment:
Shouldn't this be `eval_result['test/ssim'] > self.test_target_value` (as done in the validation goal)?